internal/controlplane/tasks.go

```
@@ -16,48 +16,50 @@ import (
+	taskConcurrencyBlockedReason = "Waiting for an available Task concurrency slot."
 )
 
 func normalizeTitleKey(value string) string {
 	return strings.ToLower(strings.TrimSpace(value))
 }
 
-func normalizeMigratedRoutineTitleKeys(ctx context.Context, tx *sql.Tx) error {
+// normalizeMigratedTaskTitleKeys runs inside migration 027, where Tasks were
+// still stored in the routines table. Migration 030 renames it to tasks.
+func normalizeMigratedTaskTitleKeys(ctx context.Context, tx *sql.Tx) error {
 	rows, err := tx.QueryContext(ctx, `SELECT id, name FROM routines WHERE migration_only = 0`)
```

golangci-lint (2.12.2)

[error] rows.Err must be checked

(rowserrcheck)